python/ray/worker.py

```diff
@@ -706,7 +726,17 @@ def _store_outputs_in_objstore(self, objectids, outputs):
                 passed into this function.
         """
         for i in range(len(objectids)):
-            self.put_object(objectids[i], outputs[i])
+            if isinstance(outputs[i], ray.actor.ActorHandleParent):
+                raise Exception("Returning an actor handle from a remote "
+                                "function is currently not allowed).")
+
+            output = outputs[i]
```

Comment:
I see, so the actor handles are being packed along with each return value, is that right?

Reply:
Yes, if there was an actor handle that was passed into the task.